03LessonThree/in-class-project/daily-planet.html

```diff
@@ -1,10 +1,54 @@
 <!DOCTYPE html>
 <html>
     <head>
+
         <title>The Daily Planet</title>
+
         <link rel="stylesheet" href="css/daily-planet-style.css"/>
+
     </head>
+
     <body>
-        <!-- You code start here -->
+
+      <header>
+        <h2>The Daily Planet</h2>
+

```

Comment:
great job here! well organized and very well spaced out to make it legible.  :dancer: